### PR TITLE
CLIO-35426: Add icon support to thAutocomplete.

### PR DESCRIPTION
**Pull request by dtomalty**

### CODE REVIEW

File: themis_components/thAutocomplete/index.scss [hunk 1]
```diff
@@ -100,11 +103,11 @@ th-autocomplete {
     }
 
     .caret {
-      display: inline-block;
+      display: none;
       width: 0;
```
Comment (JeffMarvin) [inline themis_components/thAutocomplete/index.scss]: These other properties can be ommitted if this is not being displayed.

File: themis_components/thAutocomplete/index.scss [hunk 2]
```diff
@@ -1,6 +1,9 @@
 th-autocomplete {
   @import "../../node_modules/ui-select/dist/select";
 
+  display: block;
+  position: relative;
+
```
Comment (JeffMarvin) [inline themis_components/thAutocomplete/index.scss]: Are these here so you can do the absolute positioning on .th-autocomplete-icon? (Just wondering.)

File: themis_components/thAutocomplete/index.scss (hunk 1 above)
Comment (dtomalty) [inline themis_components/thAutocomplete/index.scss]: Good call :)

File: themis_components/thAutocomplete/index.scss (hunk 2 above)
Comment (dtomalty) [inline themis_components/thAutocomplete/index.scss]: Exactly. The positioning of the child icon element doesn't work without this.

File: themis_components/thAutocomplete/index.scss [hunk 3]
```diff
@@ -100,17 +103,7 @@ th-autocomplete {
...
-      height: 0;
-      margin-left: 2px;
-      
-      vertical-align: middle;
-
-      border-top: 4px dashed;
-      border-top: 4px solid \9;
-      border-right: 4px solid transparent;
-      border-left: 4px solid transparent;
+      display: none;
```
Comment (seanhealy) [inline themis_components/thAutocomplete/index.scss]: If an icon isn't specified do we still get / want the `^`?

Comment (dtomalty) [inline themis_components/thAutocomplete/index.scss]: Spoke with design, they don't want the `^`. Logic was `^` gives the impression that it's something you can click and expect to see a drop down, which is not the case with autocomplete. The drop down reveals itself when you start typing, so doesn't need the indicator.

File: themis_components/thAutocomplete/index.scss (hunk 1 above)
Comment (JeffMarvin) [inline themis_components/thAutocomplete/index.scss]: :white_check_mark: Resolved

File: themis_components/thAutocomplete/index.scss (hunk 2 above)
Comment (JeffMarvin) [inline themis_components/thAutocomplete/index.scss]: :white_check_mark: Resolved

File: themis_components/thAutocomplete/index.scss (hunk 3 above)
Comment (JeffMarvin) [inline themis_components/thAutocomplete/index.scss]: :white_check_mark: Resolved

**Pull request merged**